orion-server/src/main/java/com/pinterest/orion/server/api/ClusterManagerApi.java

```diff
@@ -111,6 +115,33 @@ public Map<String, Map<String, Utilization>> getUtilizationDetailsByCluster() {
     return utilizationMap;
   }
 
+  @Path("/describeImages")
+  @GET
+  public List<Ami> describeImages(
+      @QueryParam("release") String os,
```

Comment:
Why not replace all these strings with the introduced variables?

Reply:
Right! Done, thanks.